webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -166,6 +161,15 @@ def test_home(self):
             ok_(link['url'] in response.content)
             ok_(link['label'] in response.content)
 
+        # The access should disappar if you cease to be active
+        User.objects.filter(id=user.id).update(is_active=False)
```

Comment:
Can you not do ``user.update(is_active=False)`` instead? You already have the ``user`` object, why do you need to get it again?